src/Shared/PackageManagers/BaseProjectManager.cs

```diff
@@ -45,7 +44,7 @@ public BaseProjectManager(string destinationDirectory = ".") : this(new DefaultH
         /// <summary>
         /// The <see cref="IHttpClientFactory"/> for the manager.
         /// </summary>
-        public IHttpClientFactory HttpClientFactory { get; }
+        public IHttpClientFactory HttpClientFactory { get; init; }
```

Comment:
This change probably wasn't needed - ctors should be refactored so it isn't needed.